Automattic/jetpack
# Add Jetpack AI forms section to Forms landing page

Fixes pcsBup-QA-p2#comment-513


## Proposed changes:

* Add a new "AI Forms" section to the Forms landing page. The CTA links to a new post if you already have AI enabled, or to the pricing page otherwise.

Desktop | Mobile
--- | ---
<img width="1691" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/088cb43f-2ae4-4c64-b3b2-510715bba0e6"> | <img width="389" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/5c4b5b54-69a7-459e-ba78-0da2fb219e9a">



### Other information:


## Jetpack product discussion


pcsBup-QA-p2#comment-513

## Does this pull request change what data or activity we track or use?


Yes. It adds a few Tracks events to the page

## Testing instructions:






* Spin up a new site using, and point to this branch using Jetpack Beta (or run it )
* Visit `/wp-admin/edit.php?post_type=feedback`
* Click on "View" on the top right corner, then on "Inbox" 
<img width="293" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/1b784b58-1461-489e-b59f-a257d37ec3b0">

* Confirm that you're redirected to `/wp-admin/admin.php?page=jetpack-forms#/landing`
* Check the copy for typos, the design, and the CTA
  * If you have AI, the CTA will link to the post creation page with the Forms block open
  * If not, you'll be redirected to the pricing page
